src/Handlebars.Net.Helpers/Helpers/StringHelpers.cs

```diff
@@ -412,6 +412,24 @@ public bool IsNotNullOrEmpty(string value)
         return !string.IsNullOrEmpty(value);
     }
 
+    [HandlebarsWriter(WriterType.String)]
+    public string Substring(string value, int start, int end = default)
+    {
+        Guard.NotNull(value);
+
+        if (value == string.Empty && start > 0)

```

Comment:
Maybe you can just skip this check and let `value.Substring` just check + throw exception.

Reply:
Yes, makes more sense.